django/db/migrations/autodetector.py

```diff
@@ -1179,6 +1180,9 @@ def generate_altered_fields(self):
 
     def create_altered_indexes(self):
         option_name = operations.AddIndex.option_name
+        self.renamed_index_together_values = defaultdict(list)
+        self.renamed_fields = {}  # FIXME '_get_altered_foo_together_operations'
```

Comment:
One way of fixing this, I think, would be to split the `generate_renamed_fields`, like the index logic, into two separate methods:
1. `create_renamed_fields`, which will populate the `self.renamed_fields` which can then be reused there
2. `generate_renamed_fields` that leverages the existing `self.renamed_fields`

What do you think?

Reply:
Tried implementing this :)